sdk/core/core-auth/test/index.spec.ts

```diff
@@ -2,8 +2,18 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 import assert from "assert";
+
+import { AzureKeyCredential } from "../src/azureKeyCredential";
```

Comment:
would it be good to add a test to make sure the rotation of the key works as expected?